.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,30 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: yarn install
+        run: yarn install
+      - name: lint
+        run: npm run lint
```

Comment:
What's the rationale behind linting before tests? I'm curious if this is needed, since we have a pre-commit hook to enforce linting (or at least we should lol).

Reply:
I did not have any hooks when committing to the fork.
How would those pre-commit hooks be called?

The logic behind this, in general, is to make sure no commits are ever made that go against the linting rules. So if the pre-commit hook takes care of it, this could be removed.

Reply:
[This pre-commit hook](https://github.com/draftbit/twitter-lite/blob/0fb4573c3d057d0ad4e9eda8fcaf9a7f4df4dcfc/package.json#L54) should enforce linting. Can you double-check that it does on your end (so we can remove the lint step from the action)?

Reply:
Yes, it runs, but e.g. when using GitHub Desktop the changes are applied after the commit has finished, meaning I could commit without having linted correctly. ( [demonstration](https://github.com/T0TProduction/twitter-lite/commits/test-if-prehook-commit-runs) )

Another thing: I'm not sure if there are rules like this in this project specifically, but in general they exist:
Non-auto-fixable issues, e.g. a rule to not allow `for...in` but force `forEach`.
In this case, the commit will go through and just not be fixed, correct?

Leaving the linting in the CI will ensure that
- If wrongfully forgetting to commit eslint fixes, the CI will fail ( and remind the author to check again )
- If committing something that is not auto-fixable, it will also fail ( making sure only desired code can be merged )

I personally think leaving linting in will make the PR reviewing process easier, as one can fully rely on the code being correctly linted as required.
It will only take about 5 seconds longer to finish the CI than without it too, so no real harm is done there.

Reply:
Maybe an important addition: In contrast to the prehook this will not actually fix the linting. It's only supposed to fail if it's not correctly linted. 
Using the prehook is still useful to make sure contributors have an easy time applying the eslint rules.